app/scripts/lib/setupSentry.js

```diff
@@ -146,31 +154,125 @@ export default function setupSentry({ release, getState }) {
       } else {
         return null;
       }
-      return breadcrumb;
+      const newBreadcrumb = removeUrlsFromBreadCrumb(breadcrumb);
+      return newBreadcrumb;
     },
   });
 
-  function rewriteReport(report) {
-    try {
-      // simplify certain complex error messages (e.g. Ethjs)
-      simplifyErrorMessages(report);
-      // modify report urls
-      rewriteReportUrls(report);
-      // append app state
-      if (getState) {
-        const appState = getState();
-        if (!report.extra) {
-          report.extra = {};
-        }
-        report.extra.appState = appState;
+  return Sentry;
+}
+
+/**
+ * Receives a string and returns that string if it is a
+ * regex match for a url with a `chrome-extension` or `moz-extension`
+ * protocol, and an empty string otherwise.
+ *
+ * @param {string} url
+ * @returns String
+ */
+function hideUrlIfNotInternal(url) {
+  const re = /^(chrome-extension|moz-extension):\/\//u;
+  if (!url.match(re)) {
+    return '';
+  }
+  return url;
+}
+
+/**
+ * Receives a Sentry breadcrumb object and potentially removes urls
+ * from its `data` property, it particular those possibly found at
+ * data.from, data.to and data.url
+ *
+ * @param {object} breadcrumb - A Sentry breadcrumb object: https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/
+ * @returns Object - A modified Sentry breadcrumb object.
```

Comment:
It looks like you're using the wrong syntax for the `@returns` part of each of these doc blocks. It should look like this:

```suggestion
 * @returns {object} A modified Sentry breadcrumb object.
```

The type goes in curly brackets. There isn't any dash because the dash is for separating the _name_ from the description, and returns don't have names.

Reply:
fixed in f183c5f2f